pkg/controller/secret/secret_controller_test.go

```diff
@@ -297,13 +299,13 @@ func Test_createPagerdutyRoute(t *testing.T) {
 }
 
 func Test_createPagerdutyReceivers_WithoutKey(t *testing.T) {
-	assertEquals(t, 0, len(createPagerdutyReceivers("", "")), "Number of Receivers")
+	assertEquals(t, 0, len(createPagerdutyReceivers("", "", "fake-cluster-id")), "Number of Receivers")
```

Comment:
In this instance, all three strings should be empty

Reply:
done